executor/analyze.go

```diff
@@ -748,6 +892,18 @@ func (e *AnalyzeColumnsExec) buildSamplingStats(ranges []*ranger.Range, needExtS
 		}
 	}
 
+	colLen := len(e.colsInfo)
+
+	indexPushedDownResult := <-idxNDVPushDownCh
+	if indexPushedDownResult.err != nil {
+		return 0, nil, nil, nil, nil, indexPushedDownResult.err
+	}
+	for _, offset := range indexesWithVirtualColOffsets {
+		ret := indexPushedDownResult.results[e.indexes[offset].ID]
+		rootRowCollector.NullCount[colLen+offset] = ret.Count
```

Comment:
why using the `ret.Count` as `NullCount`?

Reply:
We don't collect anything except the FMSketch and null count in the request. So we use the `ret.Count` directly.